docs/APISocketOptions.md

```diff
@@ -722,10 +725,10 @@ By default this value is set to 0, which means that this mechanism is off.
 | -------------------- | ----- | ------- | ---------- | ------- | -------- | ------ | --- | ------ |
 | `SRTO_MAXBW`         | 1.0.5 | pre     | `int64_t`  | B/s     | -1       | -1..   | RW  | GSD    |
 
-- Maximum send bandwidth.
-- `-1`: infinite (the limit in Live Mode is 1 Gbps)
-- ` 0`: relative to input rate (see [`SRTO_INPUTBW`](#SRTO_INPUTBW)) 
-- `>0`: absolute limit in B/s
+- Maximum send bandwidth:
+  - `-1`: infinite (the limit in Live Mode is 1 Gbps);
+  - ` 0`: relative to input rate (see [`SRTO_INPUTBW`](#SRTO_INPUTBW));
+  - `>0`: absolute limit in B/s.
```

Comment:
B/s = bits/s?

Reply:
Bytes per second.

Reply:
I think it could be mentioned in the beginning, before the whole option table, that all options that refer to transmission speed or bandwidth limit, are usually expressed in bytes per second, denoted as B/s.

Reply:
Added to the legend:
> - `B/s` - bytes per second.